var/spack/repos/builtin/packages/grass/package.py

```diff
@@ -195,3 +195,13 @@ def configure_args(self):
             args.append('--without-geos')
 
         return args
+
+    # see issue: https://github.com/spack/spack/issues/11325
+    # 'Platform.make' is created after configure step
+    # hence invoke the following function afterwards
+    def fix_iconv_linking(self):
+        makefile = FileFilter('include/Make/Platform.make')
+        makefile.filter('^ICONVLIB\s*=\s*', 'ICONVLIB = -liconv')
```

Comment:
Change this to use a raw string so flake doesn't complain about the escape sequences not being valid python.

`makefile.filter(r'^ICONVLIB\s*=\s*', 'ICONVLIB = -liconv')`